### PR TITLE
Discuss Pluto

### LINKED ISSUE #1: need more detail on pluto
It may not be a planet, but we still care

### PR DESCRIPTION
**Pull request by arblastertps**
Please see my suggestions. Fixes #1 :D

### CODE REVIEW

**Comment by arblastertps** [comment]
Fixes #1

File: pluto.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Some text about the dwarf planet Pluto to contribute to that issue
```
Comment (AliceSLee) [inline pluto.txt:1]: great input! such wow

**Review by AliceSLee — APPROVED** [review]

**Review by AliceSLee — CHANGES_REQUESTED** [review]
Test: already approved this input - is it possible to still request changes?

File: pluto.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 Some text about the dwarf planet Pluto to contribute to that issue
+It's a lumpy rock. No atmosphere, so it gets hot-ish in the sun
```
Comment (AliceSLee) [inline pluto.txt:2]: good to know! I'll leave my warm blanket at home

**Review by AliceSLee — APPROVED** [review]
Sufficient information about Pluto received

File: pluto.txt (hunk 2 above)
Comment (arblastertps) [reply pluto.txt:2]: Not recommended! Please keep warm blanket for the journey

**Pull request merged**